autogluon/task/object_detection/pipeline.py

```diff
@@ -260,6 +260,7 @@ def train(net, train_data, val_data, eval_metric, ctx, args, reporter, final_fit
             else:
                 raise NotImplementedError('%s not implemented.' % args.meta_arch)
             trainer.step(batch_size)
+            mx.nd.waitall()
```

Comment:
why is it required?

Reply:
it is not required, but could save memory